apps/astarte_appengine_api/lib/astarte_appengine_api/config.ex

```diff
@@ -33,6 +33,12 @@ defmodule Astarte.AppEngine.API.Config do
     type: NonNegativeInteger,
     default: 10000
 
+  @envdoc "The port used from the Phoenix server."
```

Comment:
Is this necessary? AFAIK this is configured from `releases.exs` (as all Phoenix APIs are)